input/size.go

```diff
@@ -1013,6 +1020,17 @@ func (twe *TxWeightEstimator) AddP2SHOutput() *TxWeightEstimator {
 	return twe
 }
 
+// AddOutputWithScriptLength estimates the weight of an output based on the
+// pkScript length.
+func (twe *TxWeightEstimator) AddOutputWithScriptLength(
+	scriptLen int) *TxWeightEstimator {
```

Comment:
This probably won't make it into the final PR. This is needed for adding the change output weight. And because we want to avoid deriving a change address before we actually need one, I just pass around the expected change script size. But I'll probably replace that with an actual change address type and then use that to estimate the weight.